### PR TITLE
Checkout Process

### PR DESCRIPTION
**Pull request by andreimid**
Add the checkout page to the web application. The form added captures all the required details to create an order on WooCommerce. The order is created via a REST API in the `pages/api/createOrder` file. Only cash payment is currently implemented. @daveGrayMac will start working on PayPal integration next. Added `success` and `failure` routes as well depending on whether the order is created successfully or not.

### CODE REVIEW

File: pages/api/createOrder.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      res.statusCode = response.status === 200 ? 200 : 201;
+      res.end();
+    } else {
+      res.statusCode = 400;
+      res.end();
+    }
+  } catch (err) {
+    console.log('Error ocurred: ', err);
+  }
+  // const json = await res.json();
```
Comment (djbotha) [inline pages/api/createOrder.js]: Might want to remove this commented code (good practice)

File: pages/checkout/failure.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Failure() {
+  return <h1>Woops, something went wrong!</h1>;
```
Comment (djbotha) [inline pages/checkout/failure.js:4]: "Oopsie whoopsie"

File: pages/checkout/index.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    width: '25ch',
```
Comment (djbotha) [inline pages/checkout/index.js:31]: Wtf is a `ch`? 😅

**Review by djbotha — APPROVED** [review]
Very nice, well done @AndreiDreyer.

File: pages/checkout/index.js (hunk 3 above)
Comment (andreimid) [reply pages/checkout/index.js:31]: ch | Relative to the width of the "0" (zero)
-- | --

Comment (djbotha) [reply pages/checkout/index.js:31]: Aight, if that works for you :D

**Pull request merged**